src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -19,20 +19,42 @@ const AdContainer = styled.section`
 
 export const getBootstrapSrc = (queryString, useLegacy = false) => {
   const adsTestScript =
-    'https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap.js';
+    'https://gn-web-assets.api.bbc.com/ngas/latest/test/dotcom-bootstrap.js';
   const adsLegacyTestScript =
-    'https://gn-web-assets.api.bbc.com/ngas/test/dotcom-bootstrap-legacy.js';
+    'https://gn-web-assets.api.bbc.com/ngas/latest/test/dotcom-bootstrap-legacy.js';
   const adsLiveScript =
-    'https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap.js';
+    'https://gn-web-assets.api.bbc.com/ngas/latest/dotcom-bootstrap.js';
   const adsLegacyLiveScript =
-    'https://gn-web-assets.api.bbc.com/ngas/dotcom-bootstrap-legacy.js';
+    'https://gn-web-assets.api.bbc.com/ngas/latest/dotcom-bootstrap-legacy.js';
+
   const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
+
   if (useLiveSrc) {
     return useLegacy ? adsLegacyLiveScript : adsLiveScript;
   }
   return useLegacy ? adsLegacyTestScript : adsTestScript;
 };
 
+export const getPreloadUrls = queryString => {
+  const useLiveSrc = isLive() || queryString.includes('ads-js-env=live');
```

Comment:
This logic is duplicated here: https://github.com/bbc/simorgh/pull/8358/files#diff-2e2eb493e4304c784acc80b54089e86784f30e11d06cdb2e1b078c98a13eaaa1R30 

Should we just pass in `isLive` as a flag to `getgetBootstrapSrc`?